package.json

```diff
@@ -39,6 +40,7 @@
   },
   "dependencies": {
     "cordova-common": "^4.0.2",
+    "cordova-js": "^6.1.0",
```

Comment:
Shouldn't this be a `devDependency`?

Reply:
Yeah, I guess I didnt reinstall it correctly on this branch.